shinywidgets/_shinywidgets.py

```diff
@@ -81,9 +82,11 @@ def init_shiny_widget(w: Widget):
     if getattr(w, "_model_id", None) is None:
         w._model_id = uuid4().hex
 
+    id = cast(str, w._model_id)  # pyright: ignore[reportUnknownMemberType]
+
     # Initialize the comm...this will also send the initial state of the widget
     w.comm = ShinyComm(
-        comm_id=w._model_id,  # pyright: ignore
+        comm_id=id,
```

Comment:
Bring in the `Widget.comm.klass` monkeypatching to happen tightly around this assignment